gen3/bin/dcf.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/gen3setup"
+
+AWS_INPUT_BUCKET="data-refresh-manifest"
+AWS_OUTPUT_BUCKET="data-refresh-output"
+GS_INPUT_BUCKET="replication-input"
+GS_OUTPUT_BUCKET="datarefresh-log"
+
+echo "Hello from dcf!"
+command=$1
+release=$2
+
+# Activate google service account
+if [ -f "${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json" ]; then
+  gcloud auth activate-service-account --key-file=${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json
+  export GOOGLE_APPLICATION_CREDENTIALS=${WORKSPACE}/${vpc_name}/apis_configs/dcf_dataservice/creds.json
+fi
+
+generate_aws_refresh_report() {
+  manifest_type=$3
+  manifest="$(aws s3 ls s3://$AWS_INPUT_BUCKET | grep GDC_full_sync_${manifest_type}_.*$release | awk -F' ' '{print $4}')"
+
+  if [ -z "$manifest" ]; then
+    echo "fail to download $manifest_type manifest!!!"
+    exit 1
+  fi
+
+  refresh_log=${manifest//.tsv/.txt}
+
+  echo "$refresh_log"
+
+  if [ ! -f "/tmp/$manifest" ]; then
+    aws s3 cp s3://$AWS_INPUT_BUCKET/$manifest /tmp/$manifest
+  fi
+
+  if [ ! -f "/tmp/$refresh_log" ]; then
+    aws s3 cp s3://$AWS_OUTPUT_BUCKET/$release/$refresh_log /tmp/$refresh_log
+  fi
+
+  python3 $GEN3_HOME/gen3/lib/dcf/aws_refresh_report.py aws_refresh_report --manifest /tmp/$manifest --log_file /tmp/$refresh_log
+
+}
+
+validate_aws_refresh_report() {
+  echo "Validate aws refresh"
+
+  manifest_type=$3
+  manifest="$(aws s3 ls s3://$AWS_INPUT_BUCKET | grep GDC_full_sync_${manifest_type}_.*$release | awk -F' ' '{print $4}')"
+  
+  if [ -z "$manifest" ]; then
+    echo "fail to download $manifest_type manifest!!!"
+    exit 1
+  fi
+  
```

Comment:
See the earlier comment about using /tmp/.  If what you really want to do is cache a file locally, then use `$GEN3_CACHE_DIR` instead, but it's not a big deal to just copy the file locally for a command line tool to avoid synchronization issues.